core/src/params.ts

```diff
@@ -5,7 +5,7 @@ import BigNumber from './bignumber';
 import getProvider from './provider';
 import MCD_CLIP_CALC from './abis/MCD_CLIP_CALC.json';
 import { RAY_NUMBER_OF_DIGITS } from './constants/UNITS';
-import { fetchContractsAddressesByNetwork, getSupportedCollateralTypes } from './contracts';
+import { fetchContractsAddressesByNetwork, getSupportedCollateralTypes } from './addresses';
 
 const PARAMS_CACHE = 24 * 60 * 60 * 1000;
```

Comment:
as above (also maybe out of scope)

Reply:
yes, let's not change it in this PR, since it's unrelated functionality